integration_tests/dbt_project.yml

```diff
@@ -30,23 +33,71 @@ seeds:
     +quote_columns: "{{ true if target.type == 'redshift' else false }}"
     +column_types:
       _fivetran_synced: timestamp
-      _fivetran_start: timestamp
-      _fivetran_end: timestamp
-      _fivetran_active: boolean
```

Comment:
Moved these so the column property warnings no longer make buildkite results hard to read.